include/pcmki/pcmki_sched_utils.h

```diff
@@ -32,8 +32,7 @@ GList *sort_nodes_by_weight(GList *nodes, pe_node_t *active_node,
 
 extern gboolean can_run_resources(const pe_node_t * node);
 
-extern void log_action(unsigned int log_level, const char *pre_text,
-                       pe_action_t * action, gboolean details);
+void log_action(const char *pre_text, pe_action_t *action, gboolean details);
```

Comment:
Might as well change the last arg to `bool` and change the couple places it's being called with TRUE/FALSE while you're at it.  Right now, it's called with both.

Reply:
Ah, already done two patches later.  I only looked one patch later to check on this.